static/app/components/feedback/feedbackItem/useFetchCrashReport.tsx

```diff
@@ -0,0 +1,35 @@
+import {Event, Group, Organization} from 'sentry/types';
+import {useApiQuery} from 'sentry/utils/queryClient';
+
+interface Props {
+  crashReportId: string;
+  organization: Organization;
+  projectSlug: string;
+}
+
+export default function useFetchCrashReport({
+  crashReportId,
+  organization,
+  projectSlug,
+}: Props) {
+  const eventEndpoint = `/projects/${organization.slug}/${projectSlug}/events/${crashReportId}/`;
+  const {data: eventData, isFetching: isEventFetching} = useApiQuery<Event>(
+    [eventEndpoint],
+    {retryDelay: 100, staleTime: 0}
```

Comment:
what's the reasoning for retrying?

Reply:
good question! the default is so slow. 

I can add a comment about that.